pkg/test/resourcefixture/testdata/basic/logging/v1beta1/logginglogexclusion/billingexclusion/update.yaml

```diff
@@ -15,8 +15,14 @@
 apiVersion: logging.cnrm.cloud.google.com/v1beta1
 kind: LoggingLogExclusion
 metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: exclusion-${uniqueId}
 spec:
+  resourceID: ${TEST_DEPENDENT_ORG_PROJECT_ID}
```

Comment:
Just to confirm: The resource ID for a LoggingLogExclusion must be a project ID?

Reply:
Thanks! No I think I copy-pasted to a wrong place. We should not make the changes to this test. I will revert it.
As you suggested, to fix the `Permission 'logging.exclusions.get' denied on resource (or it may not exist).` on this test, we may need to reference to `IAMPolicyMember`. 
While, I purpose we fix the non-billing account related issue in another PR.

Reply:
Reverted